lib/fluent/plugin/filter_kubernetes_log_level.rb

```diff
@@ -20,28 +20,28 @@ module Plugin
     class KubernetesLogLevelFilter < Fluent::Plugin::Filter
       Fluent::Plugin.register_filter("kubernetes_log_level", self)
 
-      # TODO - solve default values problem
-      config_param :log_level_label, :string
-      config_param :log_level_key, :string
-      config_param :default_logging_level, :string
+      config_param :log_level_label, :string, :default => 'logging-level'
+      config_param :log_level_key_label, :string, :default => 'logging-level-key'
+      config_param :default_log_level_key, :string, :default => 'level'
+      config_param :default_logging_level, :string, :default => nil
 
       def configure(conf)
           super
       end
 
       def level_to_num(level)
-        case level
-        when 'trace', 'Verbose'
+        case level.downcase
```

Comment:
These were actually inserted to handle Serilog log levels. While I do agree with downcasing them - it should be done for all cases.